### ISSUE TITLE
[packapp] Support conda install

### ISSUE THREAD

**Issue by asavaritayal** [Azure/azure-functions-core-tools#1031]
If `environment.yaml` is present, `conda install` the dependencies instead of `pip install`.

**Comment by asavaritayal** [comment]
@elprans can you look into this?

**Comment by asavaritayal** [comment]
@elprans correct me if I'm wrong but the Conda repository only offers pre-built wheels. So there is no requirement for `--build-native-deps` path on this one right?

**Comment by daguito81** [comment]
Any news on this issue?

**Comment by yubozhao** [comment]
Would love to see any update/news on this feature.

**Comment by anthonychu** [comment]
@anirudhgarg @vrdmr Do you have the latest guidance on this?

**Comment by georghildebrand** [comment]
I would also be very much interested in this!

**Comment by martikevin** [comment]
Hey @anirudhgarg @vrdmr we are also waiting for conda support because some dependencies like geopandas cannot be installed over pip install command. 

Please an update!?

**Comment by yputter** [comment]
Also waiting for this, as turbodbc fails with pip

**Comment by martinjoshua** [comment]
Would also love to see an update on this feature request.

**Comment by Servot** [comment]
Also waiting for this, as scikit-garden is failing with pip

**Comment by yubozhao** [comment]
We end up using custom Dockerfile.  You can find what we did here: https://github.com/bentoml/BentoML/blob/master/docker/azure-functions/Dockerfile

Basically, build on top this and add miniconda

**Comment by owenlamont** [comment]
I'm also holding out for this - would love Azure functions to have the same support for conda environments as Azure ML.

**Comment by stefanushinardi** [comment]
Hi @owenlamont, we are planning to publish an image with conda dependencies that you could use in custom container on a premium plan.  Would that work for you? or you're planning to use consumption plan?

**Comment by owenlamont** [comment]
Hi @stefanushinardi - I'm ignorant of what plan my workplace has. I'll have to enquire with the with our data engineering team.

I was aware conda environments could be used by specifying a custom container Docker image but I had hoped I could let Azure Functions abstract Docker away and it would work as written on the description of this ticket - that I could just replace the requirements.txt with an environment.yml and everything else would auto-magically work.

**Comment by stefanushinardi** [comment]
I see, got it. I dont have a timeline for that yet, but appreciate the nudge. Will discuss internally about the support. cc: @vrdmr @Hazhzeng @AnatoliB

**Comment by kyle-pena-nlp** [comment]
My general experience is that the conda ecosystem (base, conda-forge, etc.) seems to work a bit better in general.  I think more of the community's testing happens on these packages as opposed to pip packages.  My feeling is that adding support for an environment.yaml file is pretty critical.

**Comment by abouroubi** [comment]
Is there any news on this ?
I use conda for my environment management and I frankly don't see my self going back to venv, it's a weird choice to force user to use an environment manager.

**Comment by yuhaodai** [comment]
years passed but nothing.

**Comment by MarkTensenSgt** [comment]
would love this implemented as well!

**Comment by abouroubi** [comment]
@anthonychu any news on this ? It's been 3 years ...

**Comment by athinton** [comment]
Yes - I'm pretty new to python, but it seems that conda is the superior environment and package manager over pip and venv.  It is disappointing that we have to resort to using docker images to utilise the best tools for the job.

+1 from me to include this feature in Azure Functions.

**Comment by AaronM44** [comment]
Also looking for this feature

**Comment by espenhk** [comment]
Yep, also looking for the same feature. Love conda and rarely ever have issues with it, so it's a significant snag that you need workarounds to be able to use azure-functions with conda...

**Comment by Kolt45** [comment]
4 years in and no change. Have to assume this is profit issue for the business otherwise it is a missed opportunity for adding new projects to Azure Functions
